main.go

```diff
@@ -222,38 +241,41 @@ func (c AzureClient) deleteResourceGroups(ctx context.Context, numberOfResourceG
 			continue
 		}
 
-		locks, lerr := c.locksClient.ListAtResourceGroupLevel(ctx, groupName, "")
+		var opts managementlocks.ListAtResourceGroupLevelOperationOptions
+		locks, lerr := c.locksClient.ListAtResourceGroupLevel(ctx, id, opts)
```

Comment:
we have a default one here fwiw:

```suggestion
		locks, lerr := c.locksClient.ListAtResourceGroupLevel(ctx, id, managementlocks.DefaultListAtResourceGroupLevelOperationOptions())
```